tools/check_format.py

```diff
@@ -635,7 +729,15 @@ def checkFormatVisitor(arg, dir_name, names):
   # python lists are passed as references, this is used to collect the list of
   # async results (futures) from running checkFormat and passing them back to
   # the caller.
-  pool, result_list = arg
+  pool, result_list, owned_directories, error_messags = arg
+
+  # Sanity check CODEOWNERS.  This doesn't need to be done in a multi-threaded
+  # manner as it is a small and limited list.
+  source_prefix =  './source/';
+  full_prefix = './source/extensions/';
+  if dir_name.startswith(full_prefix) and '/' in dir_name[len(full_prefix):]:
```

Comment:
sorry to come back to this but is the 2nd check just making sure it's a subdirectory? Can you add a small comment?